f5_openstack_agent/lbaasv2/drivers/bigip/service_adapter.py

```diff
@@ -555,9 +555,6 @@ def _add_profiles_session_persistence(self, listener, pool, vip):
                     vip['fallbackPersistence'] = '/Common/source_addr'
 
             if persistence_type in ['HTTP_COOKIE', 'APP_COOKIE']:
-                if protocol == "TCP":
```

Comment:
Not quite sure why we delete this... Anyway it should not change any behavior... Approve this cherry-pick now. We can look into 7a023bba9fb733cd9b707112f6b5d7a1a889ec83 in the future to find out its expected behavior.